src/app/plan/duck/operations.ts

```diff
@@ -242,14 +242,23 @@ const putPlan = planValues => {
       dispatch(Creators.updatePlan(latestPlan));
       const updatedMigPlan = updateMigPlanFromValues(latestPlan, planValues);
 
-      const putRes = await client.put(
-        new MigResource(MigResourceKind.MigPlan, migMeta.namespace),
-        latestPlan.metadata.name,
-        updatedMigPlan
-      );
-      // TODO: Need some kind of retry logic here in case the resourceVersion
-      // gets ticked up in between us getting and putting the mutated object back
-      dispatch(Creators.updatePlan(putRes.data));
+      const fetch_retry: any = async n => {
```

Comment:
NIT: Can you update the var names here to be more descriptive, something like `i -> currentTry` and `n -> totalTries`. you could also initialize i here to be 1 and avoid incrementing it when checking isLastAttempt (`currentTry <= totalTries` too in the loop condition). Also, the rest of our local vars are camel cased, so maybe `fetchRetry` should be too.